package.json

```diff
@@ -141,7 +141,7 @@
     "ngx-base": "2.3.2",
     "ngx-bootstrap": "1.9.3",
     "ngx-fabric8-wit": "6.20.4",
-    "ngx-forge": "0.2.34",
+    "ngx-forge": "^0.3.6",
```

Comment:
Do we need ^ here?

Reply:
we should not use the caret and you will need to update the package-lock again